app/server/credential/src/main/java/io/syndesis/server/credential/OAuth1Applicator.java

```diff
@@ -46,8 +46,10 @@ public Connection applyTo(final Connection connection, final OAuthToken token) {
         final Connection.Builder mutableConnection = new Connection.Builder().createFrom(connection)
             .lastUpdated(new Date());
 
-        Applicator.applyProperty(mutableConnection, accessTokenValueProperty, token.getValue());
-        Applicator.applyProperty(mutableConnection, accessTokenSecretProperty, token.getSecret());
+        if (token != null) {
```

Comment:
Can `token` be null at this point? If so perhaps it would be better to fail rather than to ignore the `null` possibility.